src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -0,0 +1,186 @@
+            examStartEnd.startExam();
+            cy.contains('Cypress modeling exercise').should('be.visible').click();
+            modelingEditor.addComponentToModel(1);
+            modelingEditor.addComponentToModel(2);
+            modelingEditor.addComponentToModel(3);
+            cy.intercept('PUT', '/api/exercises/*/modeling-submissions').as('createModelingSubmission');
+            cy.contains('Save').click();
+            cy.wait('@createModelingSubmission');
+            cy.get('#exam-navigation-bar').find('.btn-danger').click();
+            examStartEnd.finishExam();
+            cy.login(tutor, '/course-management/' + course.id + '/exams');
+            cy.contains('Assessment Dashboard', { timeout: 60000 }).click();
+            assessmentDashboard.startAssessing();
+            modelingAssessment.addNewFeedback(2, 'Noice');
+            modelingAssessment.openAssessmentForComponent(1);
+            modelingAssessment.assessComponent(1, 'Good');
+            modelingAssessment.openAssessmentForComponent(2);
+            modelingAssessment.assessComponent(0, 'Neutral');
+            modelingAssessment.openAssessmentForComponent(3);
+            modelingAssessment.assessComponent(-1, 'Wrong');
+            assessmentDashboard.submitAssessment();
+            cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+            cy.get('.question-options').contains('2 of 10 points').should('be.visible');
+        });
+
+        it('Assess a text exercise submission', () => {
+            const exerciseTitle = 'Cypress Text Exercise';
+            courseManagementRequests.createAndAddTextExerciseToExam(exerciseGroup, exerciseTitle);
+            courseManagementRequests.generateMissingIndividualExams(exam);
+            courseManagementRequests.prepareExerciseStartForExam(exam);
+            cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+            examStartEnd.startExam();
+            cy.contains(exerciseTitle).click();
+            cy.get('#text-editor-tab').type(textSubmission.text);
+            cy.contains('Save').click();
+            cy.get('#exam-navigation-bar').find('.btn-danger').click();
+            examStartEnd.finishExam();
+            cy.login(tutor, '/course-management/' + course.id + '/exams');
+            cy.contains('Assessment Dashboard', { timeout: 60000 }).click();
+            assessmentDashboard.startAssessing();
+            assessmentDashboard.addNewFeedback(7, 'Good job');
+            assessmentDashboard.submitAssessment();
+            cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+            cy.get('.question-options').contains('7 of 10 points').should('be.visible');
+        });
+    });
+
+    describe('Exam Programming Exercise Assessment', () => {
+        let programmingExerciseName: string;
+        let programmingExerciseShortName: string;
+        const examEnd = (Cypress.env('isBamboo') ? GROUP_SYNCHRONIZATION / 1000 : 0) + 115;
```

Comment:
I think a comment would be nice to explain why different environments need different timeouts.
Also: You do multiple conversion to second milliseconds. I suggest just using milliseconds.
```suggestion
        const examEnd = (Cypress.env('isBamboo') ? GROUP_SYNCHRONIZATION : 0) + 115000;
```